Header header
string[] part
uint32[] x
uint32[] y
float64[] confidence
